theoplayer/how-to-guides/01-ads/04-yospace.md

```diff
@@ -2,299 +2,27 @@
 
 Yospace is a Server-Side Ad Insertion solution. A demo for the Web SDK can be found [here](https://demo.theoplayer.com/ssai).
 
-## SDKs
-
-| Web SDK | Android SDK | iOS SDK |  tvOS SDK  | Android TV SDK | Chromecast SDK |
-| :-----: | :---------: | :-----: | :--------: | :------------: | :------------: |
-|   Yes   |     Yes     |   Yes   | Unverified |      Yes       |      Yes       |
-
-## Pre-requirements
-
 ##### Web SDK
 
-1. Your THEOplayer SDK needs to have the `yospace` module enabled.
-2. You need to include the Yospace's JavaScript SDK as this is a dependency.
+You need to set up [THEOplayer Yospace Connector for Web](https://github.com/THEOplayer/web-connectors/tree/main/yospace). Please refer to [this](https://www.theoplayer.com/docs/theoplayer/connectors/web/yospace/) section to learn all about how to integrate and use the Yospace connector on Web SDK.
```

Comment:
I *think* you can do this, so the link will also work in the deploy preview:
```suggestion
You need to set up [THEOplayer Yospace Connector for Web](https://github.com/THEOplayer/web-connectors/tree/main/yospace). Please refer to [this](/theoplayer/connectors/web/yospace/) section to learn all about how to integrate and use the Yospace connector on Web SDK.
```
(Same thing for the other links below.)